server/controllers/appointmentsController.test.ts

```diff
@@ -109,25 +120,15 @@ describe('Appointments Controller', () => {
       body: {},
       query: {},
       path: 'appointments',
-      session: {
```

Comment:
We're no longer storing the results of the HMPPS Auth API call in `req.session` since there's minimal overhead in inspecting the token for every request compared to calling an API.